imagetest/test_suites/storageperf/iops_read_test.go

```diff
@@ -17,15 +17,11 @@ import (
 
 const (
 	commonFIOReadOptions = "--name=read_iops_test --filesize=10G --numjobs=8 --time_based --runtime=60s --ramp_time=2s --direct=1 --verify=0 --bs=4K --iodepth=256 --randrepeat=0 --rw=randread --group_reporting=1 --iodepth_batch_submit=256  --iodepth_batch_complete_max=256 --output-format=json"
-	windowsDriveLetter   = "F"
 )
 
 func RunFIOReadWindows() ([]byte, error) {
 	testdiskDrive := windowsDriveLetter + ":\\"
```

Comment:
I think windowsDriveLetter is undefined after being removed in one of these commits

Reply:
Nvm saw it in utils